BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,5 +1,18 @@
 {% extends 'BraincraftedBootstrapBundle:Form:bootstrap.html.twig' %}
 
+{% block form %}
```

Comment:
@itkg-ngilain in the case of the datetime error, there are no global errors, this way I cannot use the block `form_errors`